config/settings.yml

```diff
@@ -5,6 +5,8 @@ chef:
 doorkeeper:
   administrators:
     - rainbowdash
+email_from_address: 'Chef Notifications <notifications@chef.io>'
```

Comment:
I'm fine with leaving the IP as is. We should probably consider updating dev-vm, but this is fine for now. Thanks for the info.